bin/newman.js

```diff
@@ -29,6 +29,7 @@ program
     .option('-d, --iteration-data <path>', 'Specify a data file to use for iterations (either json or csv).')
     .option('--export-environment <path>', 'Exports the environment to a file after completing the run.')
     .option('--export-globals <path>', 'Specify an output file to dump Globals before exiting.')
+    .option('--reporter-cli-no-success-assertions', 'Show no output on successful assertions')
```

Comment:
No need to add a reporter specific option.
Newman automatically accumulates all the configurations passed to the `--reporter-` option and pass on to the relevant reporter. More info: https://github.com/postmanlabs/newman#configuring-reporters